Makefile

```diff
@@ -0,0 +1,124 @@
+# config
+MAKEFLAGS += --warn-undefined-variables
+SHELL := bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := all
+.DELETE_ON_ERROR:
+.SUFFIXES:
+
+# Run `make all` to create a new CURATO release:
+# - download the latest build of ROBOT
+# - download the latest versionos of IAO and BFO
+# - create curato-edit.owl from the template
+# - merge curato-edit.owl with imports to create curato-merged.owl
+# - reason over curato-merged.owl to create curato.owl
+# - clean build files
+
+# If you wish to keep build/robot.jar, run `make release` instead.
+
+# ===============================
+#           VARIABLES
+# ===============================
+
+SHELL   := /bin/bash
+OBO	:= http://purl.obolibrary.org/obo
+DEV	:= $(OBO)/curato/dev
+ROBOT	:= java -jar build/robot.jar
+
+# release vars
+TODAY	:= $(shell date +%Y-%m-%d)
+TS	:= $(shell date +'%d:%m:%Y %H:%M')
+
+# directories
+SRC = src/ontology
+
+
+# ===============================
+#             MAIN TASK
+# ===============================
+
+# run `make all` or `make release` to make a new release
+# `make all` will remove the build dir with ROBOT on completion
+all: clean
+
+### Directories
+#
+# This is a temporary place to put things.
+build:
+	mkdir -p $@
+
+# ===============================
+#             ROBOT
+# ===============================
+
+# download the most recent build of ROBOT
+build/robot.jar: | build
+	@echo "Getting ROBOT" && \
+	curl -L https://github.com/ontodev/robot/releases/download/v1.7.0/robot.jar > ./build/robot.jar
+
+# get IAO and BFO:
+build/bfo-iao: | build
+	@echo "Getting IAO and BFO latest versions" &&
+	curl -L http://purl.obolibrary.org/obo/bfo.owl   > ./ontology-source-files/bfo.owl 
+	curl -L http://purl.obolibrary.org/obo/iao.owl   > ./ontology-source-files/iao.owl
+
+clean: | release
+	@echo "Removing build files" && \
+	rm -rf build
+
+# ===============================
+#           CURATO TASKS
+# ===============================
+
+# generate curatO-edit from template file:
+build/template: | build/robot.jar build
+	@echo "Creating from Template $< to $@" && \
+	$(ROBOT) template --template src/curatO-curation-ontology-capabilities-all-classes-2020-09-07.csv  \
```

Comment:
The makefile looks right to me but actually fails at this step when I try to run it, with the error
`UNKNOWN ENTITY ERROR could not interpret 'curato:0000088' in row 4, column 0 ("ID") in table "src/curatO-curation-ontology-capabilities-all-classes-2020-09-07.csv".
For details see: http://robot.obolibrary.org/template#unknown-entity-error`
Is the repo definitely using the latest version of ROBOT because the error is not on the robot template page

Reply:
I faced the exact same issue @daniwelter . the build runs with the version locally build version of robot but not with robot.jar curled from the release. I am puzzled by this error.

Reply:
@daniwelter It actually worked with robot release 1.6. I wonder if it is a manifestation of  java build & cross platform issue

Reply:
@proccaserra I can confirm that I can now build using robot 1.6. Must be a bug in the latest release.